jaxrs/src/main/java/io/airlift/jaxrs/JaxrsBinder.java

```diff
@@ -49,6 +55,16 @@ public void bind(Key<?> targetKey)
         resourceBinder.addBinding().to(targetKey).in(SINGLETON);
     }
 
+    public <T> void bindRequestScoped(Class<T> requestScopeClass, Class<? extends Supplier<T>> scopedFactory)
+    {
+        factoryBinder.addBinding().toInstance(scopedFactoryBinding(requestScopeClass, scopedFactory));
+    }
+
+    public <T> void bindRequestScoped(Class<T> requestScopeClass, Supplier<T> scopedFactory)
```

Comment:
This doesn't seem to be tested. How would it work? Is this binding a singleton?

Reply:
Yeah, probably doesn't make sense. Dropped.